tests/test_util.py

```diff
@@ -361,10 +361,11 @@ def test_thaiword_to_date(self):
             now + timedelta(days=-2), thaiword_to_date("วานซืน", now)
         )
 
-        self.assertEqual(
-            thaiword_to_date("วันนี้").day + 1,
-            thaiword_to_date("พรุ่งนี้").day,
-        )
+        # it's error if "พรุ่งนี้" is 1 not 32.
+        #self.assertEqual(
+        #    thaiword_to_date("วันนี้").day + 1,
+        #    thaiword_to_date("พรุ่งนี้").day,
+        #)
```

Comment:
why do we comment these lines out?

Reply:
> 
> 
> why do we comment these lines out?

It has bug. (If day is 31 and tomorrow is 1, `thaiword_to_date("วันนี้").day + 1` is 32)

Reply:
so, we will fix it later? should we create a issue for that?

Reply:
maybe we can just skip the test. For example, we can do something similar to https://stackoverflow.com/a/16138561.

Reply:
> 
> 
> so, we will fix it later? should we create a issue for that?

Yes.